src/UserApp.ts

```diff
@@ -0,0 +1,173 @@
+import { DivsStructure } from "../utils/HtmlStructures"
+import { $, render, toLCaseIncludes, toArr } from "./variables/functions"
+
+
+// Enum with members representing buttons on the top of the app
+export enum CurrentButton {
+    Search,
+    Add,
+    Remove,
+    ShowAll
+}
+// User class
+export class User {
+    name: string
+    surname: string
+    dateOfBirth: number
+
+    constructor( name: string, surname: string, dateOfBirth: number ) {
+        this.name = name
+        this.surname = surname
+        this.dateOfBirth = dateOfBirth
+    }
+
+}
+// State class
+class UserAppState {
+    // Current button type
+    btnState: CurrentButton
+    // Array of found users
+    foundUsers: Array<User> = []
+    // Check if submit button clicked
+    ifSbmtClicked: boolean = false
+    // Check if user's input is right
+    ifExecutedRight: boolean
+    // Array of users
+    users: Array<User> = []
+    // Removed users
+    removedUser = []
+    // TODO here (constructor, function bodies etc)
+    constructor( selectedButton: CurrentButton ) {
+        this.btnState = selectedButton
+    }
+    // Add new User to array
+    addUser( name: string, surname: string, dateOfBirth: number ) {
+        this.users.push( new User( name, surname, dateOfBirth ) )
+        return this.users
+    }
+    // Put eaither found users array either users array to render body 
+    changeUsers( clickedBtn: CurrentButton ) {
+        if ( clickedBtn === CurrentButton.Search && this.ifSbmtClicked === true )
+            return this.foundUsers
+        else return this.users
+    }
+}
+
+
+
+
+const updateState = ( state: UserAppState ) => {
+    // The html structure of the app
+    let body =
+        DivsStructure( state.btnState, state.changeUsers( state.btnState ), state.removedUser, state.ifSbmtClicked, state.ifExecutedRight )
+    // Put body into the main div
+    render( body )
+
+    // Get buttons on the top of the app
+    const upperBtns = () => {
```

Comment:
This is an inline function (a function defined within another function as opposed to global scope), which is fine by itself, but its name `upperBtns` sounds like a name for the array, or some other collection. 

It's a good practice to call functions by actions they do, e.g. `getUpperButtons()`.